CBLClient/Apps/CBLTestServer-C/src/CollectionMethods.cpp

```diff
@@ -367,4 +371,53 @@ namespace collection_methods {
             write_serialized_body(conn, memory_map::store(token, nullptr));
         });
     }
+    
+    void collection_saveDocuments(json& body, mg_connection* conn) {
```

Comment:
Just for general understanding: why do we implement this function in C? I'm asking because if we do it in Python, it's abstracted and therefore doesn't need to be implemented in other platforms (iOS, Jaa, ect.).